src/test/java/org/sonarsource/scanner/maven/SonarQubeMojoTest.java

```diff
@@ -210,4 +243,17 @@ private Map<String, String> readProps(String filePath) throws FileNotFoundExcept
     }
   }
 
+  private static Set<String> extractSonarSources(String propertiesPath, Path projectBarDir) throws IOException {
+    String sources = readProps(propertiesPath)
+      .entrySet()
+      .stream()
+      .filter(e -> e.getKey().toString().equals("sonar.sources"))
```

Comment:
(Not sure if this comment is useful here): `sonar.sources` might be redefined per module. In that case, `sonar.sources` can be prefixed by the module key (ie: `sonar-maven-plugin.sonar.sources`).
```suggestion
      .filter(e -> e.getKey().toString().endsWith("sonar.sources"))
```